app.rb

```diff
@@ -1,8 +1,18 @@
 require 'sinatra'
-require 'sinatra/reloader'
+require "sinatra/reloader"
 require_relative 'db/connection'
 require_relative 'models/car'
 
 get '/cars' do
   @cars = Car.all
-end
\ No newline at end of file
+  erb :index
+end
+
+get '/cars/add' do
+  erb :add
+end
+
+post '/cars/add' do
```

Comment:
:+1: This works but, if you're to follow RESTful conventions, your route would be `post "/cars" do`